### PR TITLE
[Y_Demkiv]Tasks_21_30

### PR DESCRIPTION
**Pull request by YarynaD**

### CODE REVIEW

File: tasks/24_index-to-insert/index-to-insert.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export function getIndexToInsert(arr, num) {
+  arr.push(num);
+  arr.sort(function(a, b) {
```
Comment (IgorStetsiuk) [inline tasks/24_index-to-insert/index-to-insert.js]: you can write it such way `.sort((a, b) => a - b)`

Comment (IgorStetsiuk) [inline tasks/24_index-to-insert/index-to-insert.js]: you can write it such way `.sort((a, b) => a - b)`

Comment (IgorStetsiuk) [inline tasks/24_index-to-insert/index-to-insert.js]: you can write it such way `.sort((a, b) => a - b)`

File: tasks/23_seek-destroy/seek-destroy.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export function destroyer(arr, ...args) {
+  for (let i = 0; i < arr.length; i++) {
+    for (let j = 0; j < args.length; j++) {
```
Comment (tarasbobak) [inline tasks/23_seek-destroy/seek-destroy.js]: Args is an array so you can use includes or indexOf to check for presence of a value. Second loop will be not needed

File: tasks/25_rot13-encoder/rot13-encoder.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+export function rot13Encoder(str) {
+  return str.split('')
+      .map.call(str, function(char) {
```
Comment (tarasbobak) [inline tasks/25_rot13-encoder/rot13-encoder.js]: This solution is very interesting! But you don't need call here

File: tasks/26_average-mark/average-mark.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+export function getAverage(marks) {
```
Comment (tarasbobak) [inline tasks/26_average-mark/average-mark.js]: 👍

File: tasks/30_people-in-bus/people-in-bus.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+export function peopleInBus(Stops) {
+  let Summ = 0;
```
Comment (tarasbobak) [inline tasks/30_people-in-bus/people-in-bus.js]: do not uppercase variables. It is reserved for classed. Use only camelCase

**Comment by tarasbobak** [comment]
Overly good job. Use const by default. Do not use let until you know it will be reassigned.

**Pull request merged**